source/fe/fe_wedge_p.cc

```diff
@@ -103,14 +103,75 @@ FE_WedgePoly<dim, spacedim>::FE_WedgePoly(
 {
   AssertDimension(dim, 3);
 
-  if (degree == 1)
+  Assert(1 <= degree && degree <= 2, ExcNotImplemented())
+
+    for (double z = 0.0; z <= 1.0; z += (degree == 1 ? 1.0 : 0.5))
+  {
+    this->unit_support_points.emplace_back(0.0, 0.0, z);
+    this->unit_support_points.emplace_back(1.0, 0.0, z);
+    this->unit_support_points.emplace_back(0.0, 1.0, z);
+
+    if (degree == 2)
```

Comment:
Maybe not for now, but if you wanted to leave a comment: These points are just the tensor product of the triangle points with `z`. We could simply create a `FE_SimplexP<2>(degree)` and ask for its support points; then you don't have to repeat the code for `degree>2` and it has to be implemented in only one place.